services/web/server/src/simcore_service_webserver/director_v2/_handlers.py

```diff
@@ -47,8 +47,8 @@
 
 
 class RequestContext(BaseModel):
-    user_id: UserID = Field(..., alias=RQT_USERID_KEY)
-    product_name: str = Field(..., alias=RQ_PRODUCT_KEY)
+    user_id: UserID = Field(..., alias=RQT_USERID_KEY)  # type: ignore[literal-required]
```

Comment:
do you have some more insights why we need this ignore?

Reply:
what I usually do is add a comment that it will change with pydantic v2 such as
```python
# type: ignore[literal-required] # wait for pydantic v2
```